src/ts/multiSend/batchTx.ts

```diff
@@ -0,0 +1,33 @@
+import { EncodedTx } from "../forwarders/forwarder";
+import { MultiSend } from "../../typedContracts/MultiSend";
+
+interface TxToBatch {
+  to: string;
+  data: string;
+  revertIfFail: boolean;
+}
+export class MultiSender {
+  constructor(private readonly multiSend: MultiSend) {}
+
+  public async batch(batch: TxToBatch[]): Promise<EncodedTx> {
```

Comment:
Should have some comments on this public method, eg that it creates call data for the multisend contract.